src/app/helptext/sharing/iscsi/iscsi-wizard.ts

```diff
@@ -0,0 +1,72 @@
+import { T } from '../../../translate-marker';
+
+export default {
+    step1_label: T("Create or Choose Block Device"),
+
+    name_placeholder: T("Name"),
+    name_tooltip: T(""),
+
+    type_placeholder: T("Type"),
+    type_tooltip: T(""),
+
+    file_placeholder: T("File"),
+    file_tooltip: T(""),
+
+    filesize_placeholder: T("Filesize"),
+    filesize_tooltip: T(""),
+
+    disk_placeholder: T("Device"),
+    disk_tooltip: T(""),
+
+    dataset_placeholder: T("Pool/Dataset"),
+    dataset_tooltip: T(""),
+
+    volsize_placeholder: T("Size"),
+    volsize_tooltip: T(""),
+
+    volblocksize_placeholder: T("Block Size"),
+    volblocksize_tooltip: T(""),
+
+    usefor_placehodler: T("What are you using this for"),
+    usefor_tooltip: T(""),
+
+    step2_label: T("Portal"),
+
+    portal_placeholder: T("Portal"),
+    portal_tooltip: T(""),
```

Comment:
`Select an existing portal or choose <i>Create New</i> to configure a new portal.`

(new text)